server/api/sample_sizes.py

```diff
@@ -93,6 +93,12 @@ def validate_hybrid_manifests_and_cvrs(contest: Contest):
             f" times the number of votes allowed ({contest.votes_allowed})"
         )
 
+    if any(count.non_cvr < 0 for count in vote_counts.values()):
```

Comment:
How exactly does a `count.non_cvr < 0` mean that the CVRS contain more votes than were entered in the contest settings?

Reply:
In hybrid audits, we get two pieces of data from users:
1. The CVRs
2. The total votes reported for each candidate
In order to figure out the vote counts for the non-CVR ballots, we subtract the CVR votes from the total votes. Then we have the vote counts for both the CVR ballots and the non-CVR ballots, and we can perform the hybrid audit on each segment (ballot comparison for CVR ballots, ballot polling for non-CVR ballots).

The error that happened in this case was that the non-CVR vote count was negative, which is invalid and caused an assertion failure in the audit math.

Reply:
Got it, thanks for explaining!

Negative non-CVR count --> more CVRs than total votes ✅